utils/ip.go

```diff
@@ -0,0 +1,113 @@
+package utils
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	EnvPodIP = "POD_IP"
+
+	StorageNetworkInterface = "lhnet1"
+)
+
+// GetLocalIPv4 returns the local IPv4 address.
+func GetLocalIPv4fromInterface(name string) (ip string, err error) {
+	iface, err := net.InterfaceByName(name)
+	if err != nil {
+		return "", err
+	}
+
+	addrs, err := iface.Addrs()
+	if err != nil {
+		return "", errors.Wrapf(err, "interface %s doesn't have address\n", name)
+	}
+
+	var ipv4 net.IP
+	for _, addr := range addrs {
+		if ipv4 = addr.(*net.IPNet).IP.To4(); ipv4 != nil {
+			break
+		}
+	}
+	if ipv4 == nil {
+		return "", errors.Errorf("interface %s don't have an IPv4 address\n", name)
+	}
+
+	return ipv4.String(), nil
+}
+
+// GetIPForPod returns the IP address for the pod.
```

Comment:
Suggest `// GetIPForPod returns the IP address for the pod from the storage network first or the cluster network.`